common/static/common/js/discussion/views/discussion_topic_menu_view.js

```diff
@@ -146,27 +149,44 @@
             },
 
             // @TODO move into utils.coffee
+            getNameWidth: function(name) {
+                var test = $('<div>'),
```

Comment:
The safe template linter is complaining because `test` isn't named `$test`, you can't call `.html()` on it (because doing this to a non-jquery object is unsafe).

Rename `test` to `$test` to fix this issue, so the safe template linter knows test is jQuery..